docs/proposals/scheduling/activation-preference/lazy-activation-preference.md

```diff
@@ -0,0 +1,459 @@
+---
+title: Introduce a lazy activation preference to Policy
+authors:
+  - "@chaosi-zju"
+reviewers:
+  - "@RainbowMango"
+  - "@chaunceyjiang"
+  - "TBD"
+approvers:
+  - "@RainbowMango"
+  - "TBD"
+
+creation-date: 2024-01-30
+---
+
+# Introduce a lazy activation preference to Policy
+
+## Background
+
+In the scenario where `Policy` responsibilities are separated from `Resource` responsibilities, 
+one role is dedicated to `Policy` management, which is referred to as cluster administrator, 
+and the other role is dedicated to `Resource` management, which is referred to as user. 
+The cluster administrator would preconfigure some `Policies` for users, then the users apply their own `Resources` and 
+propagated them according to the preconfigured `Policy`.
+
+However, the administrator has requests to modify the `Policy` in the future, such as migrating propagation clusters.
+As the current `Policy` modification is effective immediately, the modification will affect the propagation status of
+a large number of resource templates and have a great impact on the system.
+
+The administrator is worried that the modification will lead to the failure of the business application in the
+non-change window, so the administrator hopes that the modification of `Policy` can be delayed until
+the business application change window takes effect.
+
+### Goals
+
+The most fundamental criterion is:
+
+* **Policy changes cannot actively cause changes to the propagation status of resource templates.**
+
+Under this criterion, a new delayed effective mechanism is introduced into `Policy`, 
+by which the referenced resource template will delay responding to the change of `Policy` until the resource template itself changes.
```

Comment:
> until the resource template itself changes.


Perhaps further clarification is needed on what exactly `changes` refers to here. Is it `resourceVersion`, or `generation`, or some other situation?

Reply:
done, I have added the description to the proposal.

> changes of resource template refer to any filed is modified except `status`, `metadata.managedFields`, `metadata.resourceVersion`, `metadata.generation`, or the label / annotation whose keys are prefixed with `.karmada.io`.